src/views/Vacancies/PageContainer.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from "react";
+
+import { NextPage } from "next";
+import VacanciesPageView from "./PageView";
+import {
+  fetchVacancyList,
+  fetchVacancyFacets,
+} from "@temp/core/apiLayer/vacancyService";
+
+export const VacanciesPageContainer: NextPage = () => {
+  const [loading, setLoading] = React.useState(false);
+  const [vacancyItems, setVacancyItems] = React.useState([]);
+  const [itemsTotal, setItemsTotal] = React.useState(0);
+  const [facetGroups, setFacetGroups] = React.useState([]);
+  const [searchFilters, setSearchFilters] = React.useState({
+    query: "",
+    facets: {},
+  });
+
+  const fetchData = async () => {
+    setLoading(true);
+    try {
+      const listResponse = await fetchVacancyList(searchFilters);
```

Comment:
Maybe use `const [listResponse, facetResponse] = await Promise.all([fetchVacancyList(searchFilters), fetchVacancyFacets(searchFilters)]);` So both calls are done in parallel.